ArticleTemplates/liveblogTemplateBlock.html

```diff
@@ -8,4 +8,7 @@
         </div>
         __BLOCK_LAST__
     </div>
+
+    __CONTRIBUTIONS_EPIC__
```

Comment:
This placeholder should go beneath div.block, as the creative will be a sibling of div.block.